crates/data_components/src/cdc.rs

```diff
@@ -31,6 +31,9 @@ pub enum CommitError {
     UnableToCommitChange {
         source: Box<dyn std::error::Error + Send + Sync>,
     },
+
+    #[snafu(display("Change committer missing"))]
```

Comment:
Need a better, more specific and self-descriptive error message.

Reply:
Resolved in new commit